### PR TITLE
optimization the image building of KubeSphere/console

### PR DESCRIPTION
**Pull request by zt1046656665**
Signed-off-by: zhoutian <zhoutian@yunify.com>

### What this PR does / why we need it:
we can build image easily by this PR.
when to webpack to get the javascript file,its will cost much time in linux/arm64, so we just get the  javascript file in linux/amd64,and then build the final images in  linux/amd64or linux/arm64.

### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:
```
optimization the image building of KubeSphere/console
```

### Does this PR introduced a user-facing change?

```release-note
first we build the pre-image to get the file by webpacking
then we can build the image of linux/amd64,linux/arm64  by copy the file
finally we will rmi the pre-image and rm the file by webpacking
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: build/Dockerfile.dapper [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 The KubeSphere Authors. All rights reserved.
```
Comment (LinuxSuRen) [inline build/Dockerfile.dapper:1]: You just created this file. So, this year should be 2022 instead of 2021.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: build/Dockerfile.dapper (hunk 1 above)
Comment (zt1046656665) [reply build/Dockerfile.dapper]: yes,it should be 2022,,thanks,i will correct this,

**Pull request merged**